pkg/controllers/namespace/namespace_controller.go

```diff
@@ -217,20 +216,8 @@ func (r *NamespaceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 			log.Error(err, "failed to build namespace and network config bindings", "Namepspace", ns)
 			return common.ResultRequeueAfter10sec, nil
 		}
-		// read anno "nsx.vmware.com/vpc_name", if ns contains this annotation, it means it will share infra VPC
```

Comment:
Line #186 and #191 also need to be changed.

Reply:
ok